### PR TITLE
add jobset cname record

### PR DESCRIPTION
**Pull request by moficodes**
[kubernetes/org#4748](https://github.com/kubernetes/org/issues/4748)

### CODE REVIEW

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 1]
```diff
@@ -519,6 +519,10 @@ kubectl.sigs:
 kueue.sigs:
   type: CNAME
   value: kubernetes-sigs-kueue.netlify.app.
+# https://github.com/kubernetes-sigs/jobset docs (@moficodes @@danielvegamyhre)
+jobset.sigs:
+  type: CNAME
+  value: kubernetes-sigs-jobset.netlify.app.
```
Comment (cblecker) [inline dns/zone-configs/k8s.io._0_base.yaml:525]:
```suggestion
  value: kubernetes-docs.netlifyglobalcdn.com.
```

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 2]
```diff
@@ -519,6 +519,10 @@ kubectl.sigs:
 kueue.sigs:
   type: CNAME
   value: kubernetes-sigs-kueue.netlify.app.
+# https://github.com/kubernetes-sigs/jobset docs (@moficodes @@danielvegamyhre)
```
Comment (cblecker) [inline dns/zone-configs/k8s.io._0_base.yaml:522]:
```suggestion
# https://github.com/kubernetes-sigs/jobset docs (@moficodes @danielvegamyhre)
```

**Comment by moficodes** [comment]
@cblecker Thanks for the feedback. Committed the suggestions. :)

**Pull request merged**